src/pages/tutorials/build-an-nft-marketplace/part-2.md

```diff
@@ -238,348 +243,348 @@ yarn install
+                                name="price"
+                                label="price"
+                                placeholder="Enter a price"
+                                variant="filled"
+                                value={formik.values.price}
+                                onChange={formik.handleChange}
+                                error={
+                                  formik.touched.price &&
+                                  Boolean(formik.errors.price)
+                                }
+                                helperText={
+                                  formik.touched.price && formik.errors.price
+                                }
+                                InputProps={{
+                                  endAdornment: (
+                                    <InputAdornment position="end">
+                                      <Button
+                                        type="submit"
+                                        aria-label="add to favorites"
+                                      >
+                                        <SellIcon /> Sell
+                                      </Button>
+                                    </InputAdornment>
+                                  ),
+                                }}
+                              />
+                            </span>
+                          </form>
+                        )}
+                      </CardActions>
+                    </Card>
+                  ))}{" "}
+              </ImageList>
+            </Fragment>
+          ) : (
+            <Typography sx={{ py: "2em" }} variant="h4">
+              Sorry, you don't own any bottles, buy or mint some first
+            </Typography>
+          )}
+        </Paper>
+      );
+    }
+    ```
 
-Explanation :
+    Explanation :
 
-- the template displays all owned NFTs. Only NFTs belonging to the logged user are selected
-- for each NFT, there is a form to make an offer at a price
-- if you do an offer, it calls the `sell` function and the smart contract entrypoint `nftContrat?.methods.sell(BigNumber(token_id) as nat,BigNumber(price * 1000000) as nat).send()`. Multiply the XTZ price by 10^6 because the smart contract manipulates mutez.
+    - The template displays all owned NFTs. Only NFTs belonging to the logged user are selected
+    - For each NFT, there is a form to make an offer at a price
+    - If you do an offer, it calls the `sell` function and the smart contract entrypoint `nftContrat?.methods.sell(BigNumber(token_id) as nat,BigNumber(price * 1000000) as nat).send()`. Multiply the XTZ price by 10^6 because the smart contract manipulates mutez.
 
-## Let's play : Sell
+### Let's play : Sell
 
 - Connect with your wallet and choose **alice** account (or one of the administrators you set on the smart contract earlier). You are redirected to the Administration /mint page as there is no NFT minted yet
```

Comment:
```suggestion
- Connect with your wallet and choose the **alice** account (or one of the administrators you set on the smart contract earlier). You are redirected to the Administration /mint page because there is no NFT minted yet
```